### PR TITLE
Automatic next release naming

### PR DESCRIPTION
**Pull request by rnorth**
Refs #91

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     ]
   },
   "jest": {
-    "collectCoverage": true,
+    "collectCoverage": false,
```
Comment (rnorth) [inline package.json]: Accidental commit - need to remove.
Coverage collection was preventing me from using debugging with jest in vs code, so I meant to temporarily disable.

File: lib/versions.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    }
+
+    const major = (lastReleaseMatch[1] - 0);
+    const minor = (lastReleaseMatch[2] - 0);
+    const patch = (lastReleaseMatch[3] - 0);
+
+    return {
+        major,
+        minor,
+        patch,
```
Comment (rnorth) [inline lib/versions.js]: TODO: Remove these - they're confusing.

**Comment by rnorth** [comment]
Tested here: https://github.com/rnorth/rd-test2/releases

**Comment by rnorth** [comment]
Hi @toolmantim, sorry for the delay, but I believe this is ready for review now.

**Comment by toolmantim** [comment]
Thanks so much @rnorth! Awesome stuff. I just merged in #105 and I'll fix the merge conflicts, review and merge when it's ready.

**Comment by rnorth** [comment]
Should I/you perhaps run prettier on this PR first, so that conflicts are limited to just genuine changes?

**Comment by toolmantim** [comment]
Hi @rnorth! I've updated the PR already with prettier, and started making a few changes. I want to bring the config names and docs in line with much of the other ones, and just make sure that as a new user it's not all too overwhelming as we add more stuff.

I'm pondering if I can remove "## Automatic Versioning Variables" altogether (and roll the examples up into the tables above) but I might look at it again with fresh eyes, and then I'll update the code to match too.

Thanks again!

**Comment by toolmantim** [comment]
👋🏼 @rnorth. Your readme additions were really well documented, but I felt the readme was starting to get really complicated with all the options Release Drafter is now supporting. So I've just pushed my take on simplifying it all (which means slightly less accurate/descriptive), whilst still hopefully providing lots of good examples so you grok what you need fairly quickly.

Could you check out the new readme param names, and changes, and let me know what you think?

**Comment by rnorth** [comment]
Hi @toolmantim, of course, that absolutely makes sense! 

I've had a quick look and I think this is much better - hits the right balance between accuracy and conciseness. Also, the placeholder names look good too.

Totally 👍 from me. Thanks for helping improve the PR so readily!

**Comment by toolmantim** [comment]
Okay, thanks @rnorth! I did some digging and made one final change… switching to [semver](https://www.npmjs.com/package/semver) for the version parsing and incrementing. I figured it's going to be the Wild West of version numbering schemes out there. `semver` supports more fuzzy matching, and we don't have to worry about maintaining/fixing that regex. I reckon this is ready to rumble!

**Pull request merged**